### PR TITLE
Slight tweaks on app utils

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: app/test_util.py [hunk 1]
```diff
@@ -50,13 +50,13 @@ def wait_for_css(self, css, timeout=5):
...
-        return WebDriverWait(self, timeout).until_not(
-            expected_conditions.presence_of_element_located((By.XPATH, xpath))
+        return WebDriverWait(self, timeout).until(
+            expected_conditions.invisibility_of_element((By.XPATH, xpath))
         )
 
     def wait_for_css_to_disappear(self, css, timeout=5):
         return WebDriverWait(self, timeout).until_not(
-            expected_conditions.presence_of_element_located((By.CSS, css))
+            expected_conditions.invisibility_of_element((By.CSS, css))
```
Comment (acrellin) [inline app/test_util.py:59]: This is `until_not` and `invisibility` -- should it be `until` above?

File: app/test_util.py [hunk 2]
```diff
@@ -69,13 +69,13 @@ def wait_for_css_to_be_clickable(self, css, timeout=5):
             expected_conditions.element_to_be_clickable((By.CSS, css))
         )
 
-    def scroll_to_element_and_click(self, element):
-        self.execute_script("arguments[0].scrollIntoView();", element)
-        return element.click()
+    def scroll_to_element_and_click(self, element, timeout=5):
```
Comment (acrellin) [inline app/test_util.py:72]: Where is the timeout being used?

File: app/test_util.py [hunk 3]
```diff
@@ -69,13 +69,13 @@ def wait_for_css_to_be_clickable(self, css, timeout=5):
...
+        self.execute_script("arguments[0].scrollIntoView(true);", element)
+        ActionChains(self).move_to_element(element).click().perform()
 
     def click_xpath(self, xpath):
         element = self.wait_for_xpath_to_be_clickable(xpath)
-        return self.scroll_to_element_and_click(element)
+        self.scroll_to_element_and_click(element)
```
Comment (acrellin) [inline app/test_util.py:78]: Shouldn't this return whatever click returns?

File: app/test_util.py (hunk 1 above)
Comment (stefanv) [reply app/test_util.py:66]: Correct!

File: app/test_util.py (hunk 3 above)
Comment (stefanv) [reply app/test_util.py]: We don't use it, but probably good practice.

**Pull request merged**